test-harness/pom.xml

```diff
@@ -73,6 +73,13 @@
       <artifactId>system-rules</artifactId>
       <version>1.19.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
why is this needed here?

Reply:
it is I simply forgot to remove it after moving it thinking I would need it.